physical_properties/pKa/example_submission_files/typeIII-MehtapIsik-1.csv

```diff
@@ -0,0 +1,50 @@
+# Submission Type III
+#
+# PREDICTION SECTION
+#
+# Fill one typeIII_macroscopic_pKas.csv template file for all predicted molecules.
+# For each molecule, report as many macroscopic pKas as your method predicts. 
+# Add new lines for each predicted molecule.
+# For molecules with multiple macroscopic pKas report each macroscopic pKa in a new line.
+# Report pKa values to two decimal places (e.g. 10.71).
+# Reporting the standard error of the mean (SEM) is optional and encouraged. If it is reported, SEM should be reported to two decimal places (e.g. 1.02).
+# For values for which you don't have an estimate, leave that cell of the csv table empty.

```

Comment:
Does this apply to both the uncertainty and the pKa? 
For example, if I can only calculate macro pkas for 5 of the molecules can I still participate? Do I have to list all of the molecule IDs with empty cells next them?

Reply:
It applies both to uncertainty and pKa.

We can handle both cases for a molecule without prediction:
1. Excluding a molecule ID line completely
2. A line with just molecule ID followed by empty cvs cells

Reply:
ok cool, I just wanted to make sure you had thought about it in a file parsing perspective.